android/java/res/layout/brave_sync_add_mobile.xml

```diff
@@ -80,14 +80,32 @@
                 android:layout_height="300dp"
                 android:layout_gravity="center"
                 android:contentDescription="@string/brave_sync_btn_laptop" />
-
         </LinearLayout>
 
+        <androidx.fragment.app.FragmentContainerView
+            android:id="@+id/brave_sync_count_down_qr"
+            android:name="org.chromium.chrome.browser.settings.BraveSyncCodeCountdownFragment"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+
+            android:layout_marginStart="15dp"
+            android:layout_marginEnd="15dp"
+            android:layout_marginBottom="15dp" />
+
         <TextView android:layout_width="match_parent"
             android:layout_height="0px"
             android:layout_weight="1"
             android:background="@android:color/transparent" />
 
+        <android.widget.Button android:id="@+id/brave_sync_btn_add_mobile_new_code"
+            android:visibility="gone"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Generate New Code"
```

Comment:
Should we translate the text here ?

Reply:
Yes, my bad, added string to resource but didn't update the button.
Thanks, fixed at https://github.com/brave/brave-core/pull/23051/commits/2947f2e3e1138052304f695dc2be374d90aa540f